<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Repository</class>
 <widget class="QDialog" name="Repository">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GitUI Repository</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="1">
      <widget class="QLabel" name="labelInformacja">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Select &quot;.git&quot; folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="labelOperacja">
       <property name="font">
        <font>
         <family>URW Bookman L</family>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QTreeView" name="wyswietlaczDrzewiak"/>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButtonWybierz">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButtonAnuluj">
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
